src/index.html

```diff
@@ -4,10 +4,47 @@
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="Product_img"
+        class="card-image"
+        >
+      <span class="card-name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </span>
+      <span class="card-code">
+          Product code: 195434
+      </span>
+      <div class="card-rating">
+        <div class="stars">
+          <div class="stars-item"></div>
+          <div class="stars-item"></div>
+          <div class="stars-item"></div>
+          <div class="stars-item"></div>
+          <div class="stars-item"></div>
+        </div>
+        <div class="card-review">
+            Reviews: 5
+        </div>
+      </div>
+      <div class="card-price">
+        <span class="card-text">Price:</span>
+        <span class="card-amount">$2,199</span>
+      </div>
+      <input
+        type="button"
+        value="Buy"
+        class="card-buy"
+        data-qa="hover"
+      >
```

Comment:
Why do you use tag input here?
Will be better if you use a semantic tag for button.
Use a button tag here